ee/clickhouse/queries/funnels/test/test_funnel_time_to_convert.py

```diff
@@ -78,6 +78,58 @@ def test_auto_bin_count_single_step(self):
             },
         )
 
+    def test_auto_bin_count_single_step_duplicate_events(self):
```

Comment:
This demonstrates the CH bug, will probably move this into a separate PR, since this edge case will remain flakey till its fixed upstream.

Reply:
Thinking of removing these from consideration in our time to convert funnels as a patch.